app/src/main/java/com/example/gradeassist/ui/dashboard/DashboardFragment.kt

```diff
@@ -28,4 +31,11 @@ class DashboardFragment : Fragment() {
         })
         return root
     }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        if (requestCode == Activity.RESULT_OK){
+            //TODO Add the needed onActivityResult if statements with resultCodes  needed for this fragments
+        }
+    }
```

Comment:
Wrote an override fun onActivityResult to handle needed activities for this fragment.

Reply:
This function is orphaned.  It's not currently being used, and I don't foresee it being used... since it is in a Fragment, and there's no Intent invoked from here.  Unused methods are clutter, which is technical debt, so I recommend removing this.